content/participants/ttjc@ravisaxena23/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Ravi Nandan Saxena
+type: "participant"
+name: "Ravi Nandan Saxena"
+portfolio: "http://bit.ly/ravisresume"
+github-link: "https://github.com/ravisaxena23"
+linkedin-link: "https://www.linkedin.com/in/ravisaxena23/"
+looking-for: "job"
+description: "expert at CSS/HTML intermediate in React/Node learning flutter/ReactNative"
+---
+
+Hi! I am a Ravi Nandan Saxena a Web developer let me walk you through my intro, projects and blogs.
+
+### Introduction
+
+I am an enthusiastic web developer. I am hungry to learn and grow. I like to do work in or before time but always in a systematic way. I have contributed to few open source organizations. I am a team player although I really love to be a team lead. 
+
+## Why are you taking this challenge?
+
+I need an job in December 2020.
+I want to learn and grow.
+
+## Something interesting about you?
+
+If I don't know something I love to learn it from anyone anywhere. I have watched almost all sci-fi seasons present on netflix lol.
+
+### Help I need?
+
+It would be good if someone tells me how to do auth for apps.
+
+### How to contact me?
+
+- Twitter
+- LinkedIn
+- E-mail
+- Call
+
+### Projects
+
+not a lot of projects are done. I will keep updating it with real links when done.
+
+My projects: _github link_ https://github.com/ravisaxena23/technojam-frontend
+             _github link_ https://github.com/ravisaxena23/crypto-ciphers
+
+
+#### Binder: tinder for books
```

Comment:
Also remove this project and the blog below. These are just meant to show you the format. You need to add your own projects and blog here.